osu.Game/Screens/Play/Player.cs

```diff
@@ -236,6 +237,9 @@ private void load(OsuConfigManager config, OsuGameBase game, CancellationToken c
             DrawableRuleset = ruleset.CreateDrawableRulesetWith(playableBeatmap, gameplayMods);
             dependencies.CacheAs(DrawableRuleset);
 
+            if (DrawableRuleset is IDrawableScrollingRuleset scrollingRuleset)
```

Comment:
This is a bit ad-hoc isn't it

Reply:
I...probably can just resolve the ruleset and get the scrolling info from there.

Reply:
Yeah not sure. Maybe it's fine. Just a bit hard to track what DI dependencies are available right now.

Reply:
I'm gonna leave this as-is for now. Rationale is that `IScrollingInfo` is used in a lot of places within `DrawableRuleset` so I'd expect for it to also be available here.